### DISCUSSION TITLE
Codespace doesn't start, and can't be restarted

### DISCUSSION THREAD

**Discussion by frikky** [community/community#47171] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

Hey,

I've been trying to develop in a codespace [for a bit for our project](https://github.com/Shuffle/shuffle), where we ran multiple Docker containers etc. The codespace was working entirely fine, but I had a couple of issues where it went OOM, hence I upgrade it. First to 4 Cores/8Gb RAM, and then to  8 Cores/16Gb RAM. 

The first upgrade worked fine, and was operational almost immediately, while the second one literally never worked. When starting the codespace, it opens up the window and starts loading in information, but it never finishes loading. After about 60 seconds, it jumps to the "Stopping Codespace" screen, before I have to click "Restart Codespace" to start the process once more.

![image](https://user-images.githubusercontent.com/5719530/218342985-01122aae-3669-4dcd-8a95-344b2883b918.png)

I tried downgrading again, and now it just doesn't work. I have commits from a days work on it as well, which would be nice to get a hold of. I've checked network activity from the codespace to the browser while it's loading as well, and I can't see any obvious reasons on the frontend side to why this is happening.

Has anyone else seen this issue? I tried searching here and googling around, and couldn't find anything.

Thanks

**Comment by frikky** [comment]
I further tried to export the data from it to a new branch as to not lose all the progress, which is when I found it in this state:

![image](https://user-images.githubusercontent.com/5719530/218343222-04fd6449-592e-401c-9377-d6e8dea0cc49.png)

**Reply by jamshidbehsudwal5** [reply]
What's new

**Reply by jamshidbehsudwal5** [reply]
> What's new

@jamshidbehsudwal5

**Comment by KartikM123** [comment, marked as answer]
@frikky thanks for reaching out! Do you mind sharing the name of your codespace which is in this state? I can take a closer look at the logs to figure out what's going on behind the scenes

**Reply by frikky** [reply]
Sure! 

The name is "Shuffle dev" for the shuffle/shuffle repo, and the autogenerated codespace name is "frikky-musical-giggle-v5v6xq79q72rwx"

Just tried it again, and it still doesn't seem to work

**Reply by KartikM123** [reply]
Thanks for following up! Based on our investigation, it looks like there was likely some sort of FS corruption caused by the OOM issues you had before.

Fortunately, I can still help you fetch your changes, but you'll need to open a ticket on our [Support page](https://support.github.com/contact) first since we'll need to provide account specific support. :)

**Reply by frikky** [reply]
Made the ticket - thanks! I wonder why it happened after upgrading though, as I don't think we ever went OOM on the larger instance.

Anyways - is this something that is intended to get properly fixed as well? I don't entirely trust codespaces anymore now, and ended up going back to local development again, seeing as I can debug and fix things like these

**Comment by Jiya913** [comment]
hey i am facing the same issue can anyone tell me how to resolve it
